flow/util/genReport.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+
+# make sure the working dir is flow/
+os.chdir(os.path.join(os.path.dirname(os.path.abspath(__file__)) , '..'))
+
+reportFilename = 'report.log'
+singleReportFilename = 'reports/' + reportFilename
+drcFilename = '5_route_drc.rpt'
+lastExpectedLog = '6_report.log'
+metricsLogFmt = 'gen-metrics-{}-check.log'
+metricsCheckFmt = 'metadata-{}-check.log'
+regexError = re.compile(r"^\[?err", re.VERBOSE | re.IGNORECASE)
+regexWarning = re.compile(r"^\[?warn", re.VERBOSE | re.IGNORECASE)
```

Comment:
are you using re.VERBOSE here?

Reply:
Fixed on 9947443.
This was left from an older expression.